src/app/services/provisioning-helper/provisioning-helper.coffee

```diff
@@ -5,6 +5,9 @@ angular.module 'mnoEnterpriseAngular'
     @pricedPlan = (plan) ->
       plan.pricing_type not in PRICING_TYPES['unpriced']
 
+    @hasPricedPlan = (plans) ->
```

Comment:
Are we using this anywhere?

Reply:
@NickenChuggets PR looks good once this is removed 👍